site/components/pages/Pledge/PledgeDashboard/Cards/RetirementsCard/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC, useEffect, useState } from "react";
+import LocalFireDepartmentIcon from "@mui/icons-material/LocalFireDepartment";
+import { Text } from "@klimadao/lib/components";
+import { trimStringDecimals } from "@klimadao/lib/utils";
+
+import { getRetirements, Retirements } from "lib/getRetirements";
+
+import { BaseCard } from "../BaseCard";
+import * as styles from "./styles";
+
+type Props = {
+  pageAddress: string;
+};
+
+export const RetirementsCard: FC<Props> = (props) => {
+  const [retirements, setBalances] = useState<Retirements | null>(null);
+
+  useEffect(() => {
+    (async () => {
+      const retirements = await getRetirements({ address: props.pageAddress });
```

Comment:
So fetching this client-side on mount means we can't properly server-render the balance, correct?
this should probably move to getStaticProps and be passed down

Reply:
As discussed, contract calls impact the intial page load significantly so we'll keep fetching client-side for now